.github/workflows/orchestrator.yml

```
@@ -0,0 +1,603 @@
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ github.token }}
+
+      - name: 🔍 Advanced Change Detection
+        id: changes
+        uses: dorny/paths-filter@v3
+        with:
+          filters: .github/path-filters.yml
+          list-files: json
+          list-dirs: false
```

actionlint (1.7.7)

input "list-dirs" is not defined in action "dorny/paths-filter@v3". available inputs are "base", "filters", "initial-fetch-depth", "list-files", "predicate-quantifier", "ref", "token", "working-directory"

(action)